script.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I would imagine in the future we will have more scripts. could we create and put this in a `scripts` folder? maybe `scripts/setup.sh`